test/stack_test.c

```diff
@@ -45,6 +45,8 @@ DEFINE_SPECIAL_STACK_OF_CONST(uchar, unsigned char)
 DEFINE_STACK_OF(SS)
 DEFINE_STACK_OF_CONST(SU)
 
+IMPLEMENT_COMPARISONS(char, char, "%c")
```

Comment:
I think all the basic types should go to `testutil.h` (as they did). Can you give an example where tests would then still need individual instances? Are you thinking typedef'ed types?